src/Administration/Resources/app/administration/src/app/asyncComponent/form/sw-datepicker/index.js

```diff
@@ -112,10 +112,6 @@ export default {
     },
 
     computed: {
-        flatpickrInputRef() {
```

Comment:
This is a breaking change. Please deprecate it instead. The suggestion of the AI may say otherwise.

Reply:
How would you go about deprecating this? The fix completely replaces the usage of this property, as it is faulty. I could leave this computed property in without any usage, but that may be considered a breaking change too depending on the definition...

Reply:
Leaving it there and adding a deprecated annotation is how we handle this.

```
/**
 * @deprecated tag:v6.6.0 - Will be removed, use XYZ instead
 * /
```